bin/gcs_cli.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Manipulate emulated GCS storage.
+
+# Usage: ./bin/gcs_cli.py CMD
+
+import os
+
+import click
+
+from google.auth.credentials import AnonymousCredentials
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+
+def get_endpoint_url(bucket_name):
+    base_url = os.environ["STORAGE_EMULATOR_HOST"]
+    return f"{base_url}/storage/v1/b/{bucket_name}"
```

Comment:
```suggestion
    return base_url
```

This didn't match the Python example:

https://github.com/fsouza/fake-gcs-server/blob/6a93dcc1a70c5b6022971d5f687251349ef1d109/examples/python/python.py#L24-L26

If I change it to just the `base_url`, then it continues to work.

```
app@socorro:/app$ ./bin/gcs_cli.py create test1_bucket
GCS bucket 'test1_bucket' in 'http://gcs-emulator:8001/storage/v1/b/test1_bucket' created.
app@socorro:/app$ ./bin/gcs_cli.py create test2_bucket
GCS bucket 'test2_bucket' in 'http://gcs-emulator:8001' created.
app@socorro:/app$ ./bin/gcs_cli.py list_buckets
test1_bucket	2024-04-16 13:12:38.540184+00:00
test2_bucket	2024-04-16 13:12:50.712069+00:00
```